homeassistant/components/zha/core/channels/base.py

```diff
@@ -43,6 +44,13 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+class ReportConfig(TypedDict, total=True):
+    """Matcher for the dhcp integration for required fields."""
```

Comment:
Maybe there is a better name for the class. Possibly `AttributeReportConfiguration` but that's very long...
The description also needs to be updated but I'm not sure exactly how to describe the class.

Reply:
The docstring seems to be a copy'n'paste error. 
I'm open on the naming. `AttrReportConfig` if somewhat shorter is preferred or `AttributeReportingConfigurartion` are fine

Reply:
I've gone for the short version, to help with `black` formatting.